libraries/abstractions/common_io/test/test_iot_timer.c

```diff
@@ -49,7 +49,7 @@
 /* Globals values which can be overwritten by the test
  * framework invoking these tests */
 /*-----------------------------------------------------------*/
-extern IotTimerHandle_t gIotTimerHandle;
+IotTimerHandle_t gIotTimerHandle;
```

Comment:
@vjjunnut Is this a valid use case in the test system? Are you running tests in parallel to applications that already allocate the timer?

Reply:
Test application can use the time in logging which uses these APIs - so in those cases, timer is already initialized.

Reply:
Much like in the Flash use-case, I don't really see how this was intended to work in the scenario you describe.

Most of the tests close the handle, the majority of the tests to not reset the handle to NULL when doing this. This means that any logging depending on this handle would get screwed as soon as the first test finish. 

Strictly speaking, depending on the module you aim to test while testing it sound like it could cause headache further down the line. Maybe an alternative timer should be used for the logging purpose in this case, is that an alternative?

Reply:
@qiutongs Any thought on this?